tests/utility/hash_test.cu

```diff
@@ -195,4 +195,95 @@ TEMPLATE_TEST_CASE_SIG("Static vs. dynamic key hash test",
     CHECK(hash(key) ==
           hash.compute_hash(reinterpret_cast<std::byte const*>(&key), sizeof(key_type)));
   }
+}
+
+template <typename Hash>
+__host__ __device__ bool check_murmurhash3_128_result(typename Hash::argument_type const& key,
+                                                      std::uint64_t seed,
+                                                      typename Hash::result_type expected) noexcept
+{
+  Hash h(seed);
+  return (h(key) == expected);
+}
+
+template <typename OutputIter>
+__global__ void check_murmurhash3_128_result_kernel(OutputIter result)
+{
+  int i = 0;
+
+  result[i++] = check_murmurhash3_128_result<cuco::murmurhash3_128<int>>(
```

Comment:
```suggestion
  result[i++] = check_murmurhash3_128_result<cuco::murmurhash3_128<int32_t>>(
```
size of `int`/`long` are arch-dependant, let's use `int32_t` and `int64_t` instead for clarity.